light-client/src/consensus/consensus_client.rs

```diff
@@ -384,6 +389,9 @@ impl<R: ConsensusRpc> ConsensusLightClient<R> {
 
         let should_apply_update = {
             let has_majority = committee_bits * 3 >= 512 * 2;
+            if !has_majority {
```

Comment:
is this something you meant to leave in?

Reply:
yes, updated the log message to be more clear.